app/models/movie.rb

```diff
@@ -1,2 +1,3 @@
 class Movie < ApplicationRecord
+  belongs_to :movie_genre
```

Comment:
You already defined `movie_genre` as an optional in the migration (`t.belongs_to :movie_genre`), so you should define it here as optional too:
```suggestion
  belongs_to :movie_genre, optional: true
```

**OR** make this field not nullable in the migration, please.
[Since Rails 5 it works in that way](https://bigbinary.com/blog/rails-5-makes-belong-to-association-required-by-default).